string sensor_transition
---
int64 sensor_state
